### PR TITLE
Added a comment about usage of the sycl_arm configuration

### PR DESCRIPTION
**Pull request by alistair-low**

### CODE REVIEW

**Comment by Rbiessy** [comment]
I'm not sure we should add `--incompatible_load_argument_is_label=false` to every configuration. It is required by the cross-compilation for some reason but the option was removed in bazel 0.12. So people using the sycl configuration with the latest bazel will have issues.

**Comment by alistair-low** [comment]
I believe that it's a workaround for an issue with bazel 0.11, so if we update the build instructions to use 0.12 then we could remove it.

**Comment by Rbiessy** [comment]
Alright.

**Comment by Rbiessy** [comment]
Are you splitting the PR? Can you keep `--incompatible_load_argument_is_label=false` for sycl_arm here?

**Comment by alistair-low** [comment]
I removed everything except the comment. I can squash the commits etc. and open a new PR

**Comment by Rbiessy** [comment]
Ok, it also works if you squash and push force on your branch.

**Review by Rbiessy — APPROVED** [review]
I can now squash and merge actually if you're happy with that?

File: tools/bazel.rc [hunk 1]
```diff
@@ -55,11 +55,13 @@ build:sycl_asan --copt -DGPR_NO_DIRECT_SYSCALLS --linkopt -fPIC
...
 build:sycl_asan --copt -DDISABLE_SKINNY=1
 
+# This configuration is for a cross-build by default.
+# For a native build on ARM, specify --cpu=arm on the command line.
 build:sycl_arm --crosstool_top=@local_config_sycl//crosstool:toolchain
 build:sycl_arm --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
 build:sycl_arm --define=using_sycl=true --define=using_trisycl=false
-build:sycl_arm --cpu=armeabi
 build:sycl_arm --incompatible_load_argument_is_label=false
+build:sycl_arm --cpu=armeabi
```
Comment (lukeiwanski) [inline tools/bazel.rc]: Is this really needed? Does the order matter in that instance?

Comment (alistair-low) [reply tools/bazel.rc]: It was accidental - feel free to revert it

**Comment by alistair-low** [comment]
@Rbiessy Yes, that would be helpful - thanks!

File: tools/bazel.rc (hunk 1 above)
Comment (Rbiessy) [reply tools/bazel.rc]: We can't really revert it unless we do another PR and another commit. If you don't have time to do it I'll just leave it as it is.

**Pull request merged**